### ISSUE TITLE
Add previous selector line number to message of no-duplicate-selectors

### ISSUE THREAD

**Issue by YodaDaCoda** [stylelint/stylelint#2303]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

`no-duplicate-selectors` produces an error on the second (and subsequent) instance of a selector, with no reference to the first. This is good, but the message would be more useful if it included the line number of the first instance, particularly when working with large files.

> Which rule, if any, is this issue related to?

`no-duplicate-selectors`

> What CSS is needed to reproduce this issue?

```css
.foo {
  color: pink;
}

.foo {
  color: green;
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "no-duplicate-selectors": true,
  }
}

```

> Which version of stylelint are you using?

`7.7.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

Atom package

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No

> What actually happened (e.g. what warnings or errors you are getting)?

I get the message `Unexpected selector ".foo"`

> What did you expect to happen?

A better message would be `Unexpected selector ".foo" (first instance at line 1)`

**Comment by jeddy3** [comment]
@YodaDaCoda Thank for the suggestion.

I agree that the user experience of this rule can be improved. Would you like to investigate the viability of making such a change to the rule, and sending in a PR?

**Comment by YodaDaCoda** [comment]
I've had a look, and I really don't understand much of what the code is doing. I'll have a look, but I'm not at all confident I'll be able to produce a workable PR.

**Comment by evilebottnawi** [comment]
@jeddy3 what is best error message? `Unexpected selector ".foo" (first instance at line 1)` ?

**Comment by jeddy3** [comment]
Dunno. Not brackets as those are reserved for rule names. Perhaps?:

`Unexpected selector ".foo" is also at line 1 (no-duplicate-selectors)`

**Comment by evilebottnawi** [comment]
@jeddy3 sgtm

**Comment by hudochenkov** [comment]
Done in #2415
